docs/configuration.md

```diff
@@ -796,18 +743,6 @@ var config = {
 >
 > ##
 >
-> > ### config.playback.loop
```

Comment:
why did you move loop from playkit to kaltura player?

Reply:
since it affected by the `PLAYBACK_ENDED` event